test/arrayops.jl

```diff
@@ -904,8 +901,7 @@ end
 end
 
 @testset "single multidimensional index" begin
-    let
```

Comment:
Is this `let` block redundant? (None of `a`, `a2`, `b`, or `I` occur in the enclosing scope?)